awx/ui_next/testUtils/enzymeHelpers.test.jsx

```diff
@@ -146,11 +146,8 @@ describe('waitForElement', () => {
     } catch (err) {
       error = err;
     } finally {
-      expect(error).toEqual(
-        new Error(
-          'Expected condition for <#does-not-exist> not met: el => el.length === 1'
-        )
-      );
+      expect(error.message).toContain('Expected condition for <#does-not-exist> not met');
+      expect(error.message).toContain('el.length === 1');
```

Comment:
Measuring the coverage of utils adds some additional things to the error messages:
```
  + [Error: Expected condition for <#does-not-exist> not met: el => {
    +   /* istanbul ignore next */
    +   cov_1mhuctepb7.f[16]++;
    +   cov_1mhuctepb7.s[27]++;
    +   return el.length === 1;
    + }]
```
These changes are to work around that while still keeping the assertions valid.